linera-base/src/data_types.rs

```diff
@@ -622,7 +622,7 @@ mod tests {
             Amount(10_000_000_000_000_000_000),
             Amount::from_str("10").unwrap()
         );
-        assert_eq!("10.", Amount(10_000_000_000_000_000_000).to_string(),);
```

Comment:
Surprised `cargo fmt` didn't get this one 🤔

Reply:
I think it's possible to write a macro that requires it, or for which it makes a difference, so I guess `cargo fmt` is careful with them. :shrug: